xformers/ops.py

```diff
@@ -11,6 +11,8 @@
 
 import torch
 
+from xformers.triton.flash_attention import _flash_attention as triton_flash
```

Comment:
This adds a strong dependency on triton to xformers, which isn't the case yet. Can you make it optional?

Reply:
Fixed this!